config/routes/routes.yaml

```diff
@@ -161,6 +161,20 @@ api_clinics_show:
         _format: json
     methods: GET
 
+# api_clinics_show_veterinaries:
+#     path: api/v1/clinics/{id}/veterinaries
+#     controller: App\Controller\ApiClinicController::showVeterinariesOfClinic
+#     defaults:
+#         _format: json
+#     methods: GET
+
+api_clinics_show_veterinaries:
+    path: api/v1/clinics/{id}/addresses
+    controller: App\Controller\ApiClinicController::showAddress
```

Comment:
I'd recommend that you use PHP namespaces properly `App\Controller\Api\ClinicController::showAddress` instead of `App\Controller\ApiClinicController::showAddress`

Reply:
Ok, done!

Reply:
Nice. Thx!